src/main/java/games/strategy/debug/ClientLogger.java

```diff
@@ -47,4 +47,15 @@ public static void logError(final String msg, final Collection<? extends Throwab
     logError(msg);
     throwables.forEach(ClientLogger::logError);
   }
+
+  public static FatalErrorException fatal(final String error) {
+    logError("Fatal error: " + error);
+    return new FatalErrorException(error);
+  }
+
+  public static class FatalErrorException extends RuntimeException {
```

Comment:
NBD, but... Static analysis: "The serializable class FatalErrorException does not declare a static final serialVersionUID field of type long"